Utils/error_loggers.js

```diff
@@ -1,9 +1,14 @@
 const Config = require('../config.js')
-var bugsnag = require('bugsnag')
+const Raven = require('raven')
 
-bugsnag.register(Config.discord.bugsnag)
+Raven.config(Config.discord.sentry).install()
 
-exports.log = function (bot, cObj, fullErr) {
-  if (fullErr !== undefined) bugsnag.notify(fullErr)
-  bot.Channels.find((c) => c.name === 'bot-error').sendMessage(`Encountered an error while trying to run ${cObj.cause}.\nReturned error: \`\`\`${cObj.message}\`\`\``)
-}
+module.exports = {
+  log: (bot, cObj, fullErr) => {
+    if (fullErr !== undefined) Raven.captureException(fullErr)
+    bot.Channels.find((c) => c.name === 'bot-error').sendMessage(`Encountered an error while trying to run ${cObj.cause}.\nReturned error: \`\`\`${cObj.message}\`\`\``)
+  },
+  raven: (e) => {

```

Comment:
you could add an "extra" param, which will get pushed to sentry for additional info
```js
raven: (error, extra) => Raven.captureException(error, { extra });
```